Cargo.toml

```diff
@@ -25,10 +25,10 @@ rand = "0.8.1"
 tempfile = "3.1.0"
 camino = "1.0.5"
 libc = "0.2.100"
-io-lifetimes = "0.4.4"
+io-lifetimes = "0.5.1"
 
 [target.'cfg(not(windows))'.dev-dependencies]
-rustix = "0.32.0"
+rustix = "0.33.0"
```

Comment:
Ahh nice and this is what I need for https://github.com/coreos/rpm-ostree/pull/3374#issuecomment-1025818269

(Seems like the commit message/PR title should mention rustix too, as I doubt I'm the only one who will be using *both* cap-std and rustix and hence wanting a single version of both)

Reply:
I edited the title to mention rustix via the squash-and-merge UI.